lib/modules/dosomething/dosomething_signup/includes/dosomething_signup.inc

```diff
@@ -95,8 +95,8 @@ class SignupEntityController extends EntityAPIController {
     // If the entity uid doesnt belong to current user:
     if ($entity->uid != $user->uid) {
       // And current user can't edit any reportback:
-      if (!user_access('edit any signup')) {
-        watchdog('dosomething_signup', "Attempted uid override for @entity by User @uid", 
+      if (!user_access('edit any signup') && !drupal_is_cli()) {
```

Comment:
yep this does work, @sergii-tkachenko did the same thing for reportback saves here https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_reportback/includes/ReportbackController.php#L106